### PR TITLE
acpi, flatpak, i3, iptables: add Chinese translation

### PR DESCRIPTION
**Pull request by whriedplanck**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by whriedplanck** [comment]
All done.

File: pages.zh/linux/acpi.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# acpi
+
+> 显示电池状态或热量信息。
+> 更多信息参见：<https://sourceforge.net/projects/acpiclient/files/acpiclient/>.
```
Comment (navarroaxel) [inline pages.zh/linux/acpi.md:4]:
```suggestion
> 更多信息：<https://sourceforge.net/projects/acpiclient/files/acpiclient/>.
```

File: pages.zh/linux/i3.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# i3
+
+> 一款动态平铺窗口管理器。
+> 更多信息参见：https://i3wm.org/docs/userguide.html>.
```
Comment (navarroaxel) [inline pages.zh/linux/i3.md:4]:
```suggestion
> 更多信息：https://i3wm.org/docs/userguide.html>.
```

File: pages.zh/linux/iptables.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# iptables
+
+> 可用于配置 Linux 内核防火墙提供的过滤表、规则链和规则的程序。
+> 更多信息参见： <https://www.netfilter.org/projects/iptables/>.
```
Comment (navarroaxel) [inline pages.zh/linux/iptables.md:4]:
```suggestion
> 更多信息： <https://www.netfilter.org/projects/iptables/>.
```

File: pages.zh/linux/flatpak.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# flatpak
+
+> 构建、安装和运行 Flatpak 应用和运行时。
+
+- 运行已安装应用：
+
+`flatpak run {{name}}`
```
Comment (navarroaxel) [inline pages.zh/linux/flatpak.md:7]: Please translate the arguments for get this easier for non-English speakers (in all examples).

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @whriedplanck, welcome and thanks for your submission. I left some comments.

**Comment by whriedplanck** [comment]
Hi, @navarroaxel, thank you for reviewing. I'll change them soon.

**Comment by whriedplanck** [comment]
@navarroaxel I've had them changed ~ Thanks for suggestions! :D

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**